tutorials/releases.md

```diff
@@ -0,0 +1,312 @@
+![CRI-O logo](../logo/crio-logo.svg)
+
+Welcome to releases page! Enjoy CRI-O!
+
+- [1.18 (latest stable)](#118--latest-stable-)
+  * [1.18.0](#1180)
+    + [CentOS 8](#centos-8)
+  * [CentOS 8 Stream](#centos-8-stream)
+    + [Debian Unstable](#debian-unstable)
+    + [Debian Testing](#debian-testing)
+    + [Fedora 31 or later](#fedora-31-or-later)
+    + [Ubuntu 20.04](#ubuntu-2004)
+- [1.17](#117)
+  * [1.17.4](#1174)
+    + [CentOS 7](#centos-7)
+    + [CentOS 8](#centos-8-1)
+    + [CentOS 8 Stream](#centos-8-stream-1)
+    + [Debian Unstable](#debian-unstable-1)
+    + [Debian Testing](#debian-testing-1)
+    + [Fedora](#fedora)
+    + [Ubuntu 18.04](#ubuntu-1804)
+    + [Ubuntu 19.04](#ubuntu-1904)
+    + [Ubuntu 19.10](#ubuntu-1910)
+    + [Ubuntu 20.04](#ubuntu-2004-1)
+- [1.16](#116)
+  * [1.16.6](#1166)
+    + [CentOS 7](#centos-7-1)
+    + [CentOS 8](#centos-8-2)
+    + [CentOS 8 Stream](#centos-8-stream-2)
+    + [Debian Unstable](#debian-unstable-2)
+    + [Debian Testing](#debian-testing-2)
+    + [Fedora](#fedora-1)
+    + [Ubuntu 18.04](#ubuntu-1804-1)
+    + [Ubuntu 19.04](#ubuntu-1904-1)
+    + [Ubuntu 19.10](#ubuntu-1910-1)
+    + [Ubuntu 20.04](#ubuntu-2004-2)
+- [1.15 (deprecated)](#115--deprecated-)
+  * [1.15.4](#1154)
+    + [CentOS 7](#centos-7-2)
+    + [Debian Unstable](#debian-unstable-3)
+    + [Debian Testing](#debian-testing-3)
+    + [Fedora](#fedora-2)
```

Comment:
I think we do not have to mention it if not supported any more.

Reply:
Last week I got a question about this repo.

Reply:
yeah I think we should drop it. That is our last release of 1.15 and any new questions or users should be pointed to a newer version